src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -52,6 +56,16 @@ class SidebarScreen extends Component {
     componentDidMount() {
         Performance.markStart(CONST.TIMING.SIDEBAR_LOADED);
         Timing.start(CONST.TIMING.SIDEBAR_LOADED, true);
+
+        if (this.props.isFirstTimeNewExpensifyUser) {
+            // For some reason, the menu doesn't open without the timeout
+            setTimeout(() => {
```

Comment:
>For some reason, the menu doesn't open without the timeout

This isn't really an acceptable solution and we should try to avoid adding things to the code that we don't understand.